src/test/java/com/wikia/webdriver/common/contentpatterns/URLsContent.java

```diff
@@ -172,6 +173,7 @@
 
   //External URL
   public static final String EXTERNAL_URL = "http://www.wikia.com";
+  public static final String EXTERNAL_RUNESCAPE = "http://runescape-test.wikia.com/";
```

Comment:
are you sure you can use production wiki for this test ? Production wiki is prone to user modification. Unless you are 100% sure the data will not be modified by anyone, prepare your data on a hermetic testing wikia environment